_redirects

```diff
@@ -0,0 +1,21 @@
+# Redirects from what the browser requests to what we serve
```

Comment:
Let's put this file in another PR so we can merge this one and that the redirects ready for when we can ship!

Reply:
Left the redirects that pointed to existing articles. Moved the rest over to move-to-guides branch and opened a pull request there with a note to wait until after guides launch.